var/classes/definition_OnlineShopOrder.php

```diff
@@ -3933,24 +3933,20 @@
                  'bodyStyle' => 'padding: 10px; background-color: #d9edf7; border-color: #bce8f1 !important; color: #31708f;',
                  'datatype' => 'layout',
                  'permissions' => NULL,
-                 'childs' => 
+                 'childs' =>
                 array (
                 ),
                  'locked' => false,
               )),
-              1 => 
+              1 =>
               Pimcore\Model\DataObject\ClassDefinition\Data\Objectbricks::__set_state(array(
                  'fieldtype' => 'objectbricks',
                  'phpdocType' => '\\Pimcore\\Model\\DataObject\\Objectbrick',
-                 'allowedTypes' => 
+                 'allowedTypes' =>
                 array (
-                  0 => 'PaymentProviderQpay',
                   1 => 'PaymentProviderDatatrans',
-                  2 => 'PaymentProviderPayPal',
-                  3 => 'PaymentProviderMpay24',
-                  4 => 'PaymentProviderOGone',
-                  5 => 'PaymentProviderWirecardSeamless',
-                  6 => 'PaymentProviderHeidelPay',
+                  2 => 'PaymentProviderHeidelPay',
```

Comment:
why still heidelpay?

Reply:
It was there for historical reasons but now changed it Unzer.